modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpClientDiscoverySpiFailureTimeoutSelfTest.java

```diff
@@ -449,7 +449,7 @@ public void clientReconnectOnCoordinatorRouterFail(int srvNodes) throws Exceptio
         private Exception err;
 
         /**  */
-        @Override protected void writeToSocket(
```

Comment:
Why removed `@Override` ?

Reply:
fixed. i've reviewd code obe more time and i've veririden method instead of overloading. so doc is inherited now.
TcpDiscoveryConnectionCheckMessage is used as a stub instead of msg which was pased via parameter according to your comment:
// Replace logic routine message with a stub to update last-sent-time to avoid segmentation on
// connRecoveryTimeout.